es/utils/async-init.js

```diff
@@ -19,15 +19,14 @@ import stampit from '@stamp/it'
 import * as R from 'ramda'
 
 function asyncInit (options = {}, { stamp, args, instance }) {
-  return R.reduce(async (instance, init) => {
-    instance = await Promise.resolve(instance)
+  return stamp.compose.deepConfiguration.AsyncInit.initializers.reduce(async (instance, init) => {
```

Comment:
Why it is better to use native `reduce` instead of `ramda`.

Reply:
Let's continue this discussion in https://github.com/aeternity/aepp-sdk-js/pull/120